economics/staking_protocol/main.tex

```diff
@@ -0,0 +1,355 @@
+\begin{equation}
+    \kappa_i &=& \left(0.5 + 0.5 \cdot \frac{\min(D_i, D_{max})}{D_{max}}\right)
+\end{equation}
+
+In Phase 1, a sub-stake $i$ of size $\theta$ will receive the following subsidy, where $\Theta$ is the sum of all locked sub-stakes and $I_{max}$ is the maximum number of tokens that can be minted in a period $p$:
+
+\begin{equation}
+    \label{eq:ds}
+    ds_{\theta i,p} = \kappa_i \cdot \frac{\theta_i}{\Theta} \cdot I_{max} 
+\end{equation}
+
+In Phase 2, a sub-stake $i$ of size $\theta$ will receive the following subsidy, where $S(\infty)$ is the total number of tokens ever to be minted (roughly 3.885 billion), $S_{p-1}$ is the circulating supply in the period before $p$, and $T_{1/2}$ is the issuance decay half life:
+
+\begin{equation}
+    \label{eq:rate-max}
+    ds_{i,p} = \kappa_i \cdot \frac{\theta_i}{\Theta} \cdot \frac{\ln{2}}{T_{1/2} \cdot 365 } \cdot \left[S(\infty) - S(p-1) \right]\\
+\end{equation}
+
+\subsection{Variable issuance and the global economy}
+
+The circulating supply at a given period $p$ is the sum of the tokens received thus far by all sub-stakes (this does not include the genesis supply):
+\begin{equation}
+    dS_p= \sum_i ds_{i,p}
+\end{equation}
+
+Sub-stakes locked for a short times earn less than the maximum subsidy, thereby slowing the expansion of the entire circulating supply – tokens are only minted if they are to be distributed to node operators. The implication of any instance of $\kappa$ < 1 is that it will take longer than the provisional 5 years to mint all the tokens reserved for Phase 1. The year in which the protocol switches from the first to second phase ($T_{1\rightarrow2}$) is therefore: 
+
+\begin{equation}
+\label{phaseswitch}
+    T_{1\rightarrow2}= \frac{S_{Ph1}}{S_0 \cdot \kappa^* \cdot I_{max}}
+\end{equation}
+\\
+Where $\kappa^*$ is the average lock duration of all sub-stakes, weighted by relative size, and $S_{Ph1}$ is the total number of tokens reserved for Phase 1, excluding the genesis supply (roughly 1.83 billion). For example, if $\kappa^*$ were to equal 11 months, $T_{1\rightarrow2}$ would be  5 years and 79 days. The longest Phase 1 can last (i.e. if all sub-stakes unlock in 1 month or less, so $\kappa^* \approx 0.54$), barring changes to the protocol, is 8 years and 209 days.
+
+\begin{figure}[h!]
+    \includegraphics[width=\textwidth]{graphs/SubsidyP1.pdf}
+    \caption{Cumulative total subsidies received in \textbf{Phase 1}, for stakes unlocking in 12+ and 1 month, and configured to re-stake or withdraw subsidies. Staking rate \& average lock duration assumed to be 70\% and 0.77 respectively.}
```

Comment:
Are these just arbitrary assumptions or are they pulled from somewhere?

Reply:
They're both sort of arbitrary. Kappa* = 0.77 if there is a perfectly even distribution of lock durations from 1 month to 12 months (`kappa_avg = (1.0 + (0.5+(0.5*(1/12)))) / 2  # uniform distribution of stake lock`). A staking rate of >= 70% seems realistic given what we know about stakers in our network and our distribution mechanism, and is close to Cosmos's target participation rate. Happy to change either